src/OpeningHours.php

```diff
@@ -159,4 +171,13 @@ protected function normalizeDayName(string $day)
 
         return $day;
     }
+
+    protected function setTimezone(DateTimeInterface $date)
```

Comment:
I think `applyTimezone` would be a better method name. `setTimezone` confused me at first since I thought it was setting a timezone on the `OpeningHours` object itself.

I'd say add a `setTimezone` method that sets the `OpeningHours` instance's timezone (if you'd rather define it fluently instead of using the constructor, and rename this one to `applyTimezone`.